app/src/main/java/com/arygm/quickfix/ui/theme/Theme.kt

```diff
@@ -20,61 +20,105 @@ import androidx.compose.ui.unit.sp
+        onTertiary = TextButtonTertiary,
+        error = AccentPrimary,
+        onError = AccentSecondary,
+        onBackground = TextPrimary,
+        onSurface = TextSecondary,
+        outline = TitlePrimary,
+        surfaceVariant = ButtonQuaternary, // Fourth button color
+        onSurfaceVariant = TextButtonQuaternary, // Text color for the quaternary button,
+        onSecondaryContainer = TextDisabled,
+        tertiaryContainer = ButtonDisabled)
 
-val interFontFamily =
+private val DarkColorScheme =
+    darkColorScheme(
+        primary = DarkButtonPrimary,
+        secondary = DarkButtonSecondary,
+        tertiary = DarkButtonTertiary,
+        background = DarkBackgroundPrimary,
+        surface = DarkBackgroundSecondary,
+        onPrimary = DarkTextButtonPrimary,
+        onSecondary = DarkTextButtonSecondary,
+        onTertiary = DarkTextButtonTertiary,
+        error = DarkAccentPrimary,
+        onError = DarkAccentSecondary,
+        onBackground = DarkTextPrimary,
+        onSurface = DarkTextSecondary,
+        outline = DarkTitlePrimary,
+        surfaceVariant = DarkButtonQuaternary, // Fourth button color
+        onSurfaceVariant = DarkTextButtonQuaternary, // Text color for the quaternary button
+        onSecondaryContainer = DarkTextDisabled,
+        tertiaryContainer = DarkButtonDisabled)
+val poppinsFontFamily =
     FontFamily(
-        Font(R.font.inter_extrabolditalic, FontWeight.ExtraBold, FontStyle.Italic),
-        Font(R.font.inter_extrabold, FontWeight.ExtraBold, FontStyle.Normal),
+        Font(R.font.poppins_black, FontWeight.Black, FontStyle.Normal),
+        Font(R.font.poppins_blackitalic, FontWeight.Black, FontStyle.Italic),
+        Font(R.font.poppins_extrabold, FontWeight.ExtraBold, FontStyle.Normal),
+        Font(R.font.poppins_extrabolditalic, FontWeight.ExtraBold, FontStyle.Italic),
+        Font(R.font.poppins_bold, FontWeight.Bold, FontStyle.Normal),
+        Font(R.font.poppins_bolditalic, FontWeight.Bold, FontStyle.Italic),
+        Font(R.font.poppins_semibold, FontWeight.SemiBold, FontStyle.Normal),
+        Font(R.font.poppins_semibolditalic, FontWeight.SemiBold, FontStyle.Italic),
+        Font(R.font.poppins_medium, FontWeight.Medium, FontStyle.Normal),
+        Font(R.font.poppins_mediumitalic, FontWeight.Medium, FontStyle.Italic),
+        Font(R.font.poppins_regular, FontWeight.Normal, FontStyle.Normal),
+        Font(R.font.poppins_italic, FontWeight.Normal, FontStyle.Italic),
+        Font(R.font.poppins_light, FontWeight.Light, FontStyle.Normal),
+        Font(R.font.poppins_lightitalic, FontWeight.Light, FontStyle.Italic),
+        Font(R.font.poppins_extralight, FontWeight.ExtraLight, FontStyle.Normal),
+        Font(R.font.poppins_extralightitalic, FontWeight.ExtraLight, FontStyle.Italic),
+        Font(R.font.poppins_thin, FontWeight.Thin, FontStyle.Normal),
+        Font(R.font.poppins_thinitalic, FontWeight.Thin, FontStyle.Italic),
```

Comment:
I see you are not using all the fonts in this font family and we most likely will only use the bolder variants of this font in the future. Make sure to remove the variants that will not be used in the project to avoid clutter (and also keeping our line coverage high!)

P.S.: since I can't really comment on the `font` package. Please make sure to remove the unused fonts from the res folder

Reply:
To be fair, this is not that much of an issue. We can keep the fonts so long as we sufficiently test them and remove them at the very end of the project to clean up the code.

just opened issue #60 so that we can callback to it later.
This conversation can be resolved